public/js/logger.js

```diff
@@ -0,0 +1,41 @@
+
```

Comment:
you have the same code and the same exports in app.js
if you make a file out of it and not use the logging abilities in the app.js then remove the double code @shirbalas